packages/fleather/test/widgets/editor_test.dart

```diff
@@ -64,6 +64,8 @@ void main() {
       final finder = find.text('Paste');
       expect(finder, findsOneWidget);
       await tester.tap(finder);
+      // account for throttling of history update
+      await tester.pump(throttleDuration);
       await tester.pumpAndSettle();
```

Comment:
You can replace these two lines with `await tester.pumpAndSettle(throttleDuration);`. Please apply this anywhere applicable.